libbeat/cmd/instance/beat.go

```diff
@@ -125,7 +125,7 @@ func Run(name, version string, bt beat.Creator) error {
 }
 
 // NewBeat creates a new beat instance
-func NewBeat(name, v string) (*Beat, error) {
+func NewBeat(name, indexPrefix, v string) (*Beat, error) {
```

Comment:
I suggest if `indexPrefix` is empty, we set it to `name`. Like this the param can be set to `""`.

Reply:
will change